google/cloud/security/iam/explain/explainer.py

```diff
@@ -118,14 +118,15 @@ def GetAccessByPermissions(self, model_name, role_name, permission_name,
                 yield role, resource, members
 
     def GetAccessByMembers(self, model_name, member_name, permission_names,
-                           expand_resources):
+                           expand_resources, explain_filter):
```

Comment:
ditto: please document the parameters and the yield

You can see examples here:
http://www.sphinx-doc.org/en/stable/ext/example_google.html